integration_tests/fixtures/forseti/main.tf

```diff
@@ -41,11 +41,12 @@ module "bastion" {
 }
 
 module "forseti" {
-  source = "github.com/forseti-security/terraform-google-forseti"
+  source = "terraform-google-modules/forseti/google"
 
```

Comment:
IIUC, when this fixture is invoked it will pull the latest module available in the Terraform Registry (atm 5.0.0).  Just double checking that this is what we want.

Reply:
Reverted this change.